balloc/balloc.h

```diff
@@ -236,9 +236,13 @@ struct m0_balloc {
 	 * before the m0_format_footer, where only persistent fields allowed
 	 */
 	/** db for free extent */
-	struct m0_be_btree           cb_db_group_extents;
+	struct m0_btree             *cb_db_group_extents;
 	/** db for group desc */
-	struct m0_be_btree           cb_db_group_desc;
+	struct m0_btree             *cb_db_group_desc;
+
+	/** Root nodes for cb_db_group_extents and cb_db_group_desc btrees. */
+	uint8_t                 cb_ge_node[1024] __attribute__((aligned(1024)));
```

Comment:
Can we use a macro to define the size of node?

Reply:
Ok.